crates/mithril-oxide-sys/cpp/IR/Types.hpp

```diff
@@ -1,10 +1,39 @@
 #pragma once
 
 #include <mlir/IR/Types.h>
+#include <mlir/IR/Value.h>
+#include <mlir/IR/BuiltinTypes.h>
 
 
 namespace mithril_oxide_sys {
 
 using mlir::Type;
+using mlir::Value;
+
+using mlir::FunctionType;
+using mlir::IntegerType;
+using mlir::FloatType;
+using mlir::TensorType;
+using mlir::BaseMemRefType;
+using mlir::MemRefType;
+using mlir::RankedTensorType;
+using mlir::VectorType;
+using mlir::IndexType;
```

Comment:
Note that not all types are built-in (memref, index, and maybe also tensors and vectors?). I think those are better off on their own dialect, but for now they're fine here.

```suggestion
using mlir::BaseMemRefType;
using mlir::FloatType;
using mlir::FunctionType;
using mlir::IndexType;
using mlir::IntegerType;
using mlir::MemRefType;
using mlir::RankedTensorType;
using mlir::TensorType;
using mlir::VectorType;
```